satpy/readers/yaml_reader.py

```diff
@@ -1276,7 +1276,7 @@ def _pad_earlier_segments_area(file_handlers, dsid, area_defs):
                           fh in file_handlers]
     area = file_handlers[0].get_area_def(dsid)
     seg_size = area.shape
-    proj_dict = area.proj_dict
+    crs = area.crs if hasattr(area, 'crs') else area.proj_dict
```

Comment:
It's been a while since the crs attribute was introduced, do we really need the fallback to proj_dict here?

Reply:
If we drop the fallback then we need to force `pyproj>2.2`

Reply:
that's almost two years old, so I'm fine with it.

Reply:
Oh boy. You heard 'em kids, we're breaking everything. Let's see what happens.

Reply:
:beers: :tada: